ansible/roles/monasca/tasks/config.yml

```diff
@@ -218,6 +218,42 @@
   notify:
     - Restart monasca-log-transformer container
 
+- name: Ensuring logstash patterns folder exists
```

Comment:
@dougszumski do you need to push this upstream?

Reply:
I may as well - It was a bit of experiment for a project which may not be using it anymore, but it might be useful for someone else.